{{cookiecutter.project_name}}/.github/workflows/build.yaml

```diff
@@ -15,15 +15,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python 3.12
-        uses: actions/setup-python@v5
         with:
-          python-version: "3.12"
-          cache: "pip"
-          cache-dependency-path: "**/pyproject.toml"
-      - name: Install build dependencies
-        run: python -m pip install --upgrade pip wheel twine build
+          filter: blob:none
+          fetch-depth: 0
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@v5
```

Comment:
I used `uv` instead of `hatch` here for the sake of running `twine check` separately

Reply:
Makes sense! I think this will warn that the lockfile glob doesn’t match, so maybe disable the cache here?